### DISCUSSION TITLE
How to disable wingpanel overlapping on hover?

### DISCUSSION THREAD

**Discussion by DanyGee** [elementary/wingpanel#623] [Q&A]
Hi,
I have a nicely configured conky config that matches wingpanel indicators style, but when I hover over it in maximized mode, the conky window gets overlapped by wingpanel.
Would like to disable this hover overlapping somehow.
Can you point me where can I find the code responsible for this?
Here is a video of what I'm talking about :eyes: 
https://mega.nz/#!HAtDWK4b!Ns5ftsCjoUJSiwVeEfVgqcSkysUPuYKkyAN815hXT24

**Comment by peteruithoven** [comment]
But wouldn't interactivity be impossible if the wingpanel wouldn't be on top? 

(Side note, it's probably easier to record and share these kind of things with something like [Screen Recorder](https://appcenter.elementary.io/com.github.mohelm97.screenrecorder/))

**Comment by DanyGee** [comment]
> But wouldn't interactivity be impossible if the wingpanel wouldn't be on top?

Damn, hadn't thought about it :thinking: I'm just looking for any way to make it not disappear and still not brake the functionality of the wingpanel.

> (Side note, it's probably easier to record and share these kind of things with something like Screen Recorder)

Yeah, tried screen recording, but record didn't show what happened smoothly (strange artifacts).

**Comment by Manueljlin** [comment]
Also happens when you set plank to fill the entire side of the screen.
@DanyGee Disable flipping to fix the recording issue:
![captura realizada el 2019-03-02 09 51 27](https://user-images.githubusercontent.com/11275573/53679801-eb98c580-3cd1-11e9-8ffe-642bc5731ce9.png)

**Comment by ochornolutskyi** [comment]
@DanyGee 
hello my friend.  what if i don't have this item in nvidia settings?
